menu/menu.yaml

```diff
@@ -1,1557 +1,1561 @@
 - Name: Getting Started
   Topics:
-  - Title: Quick Start Tutorial
```

Comment:
Are the spacing changes deliberate (or necessary)?

Reply:
I didn't make them - visual studio code auto adjusts any formatting when you save a page.